src/oglGraph/GLFont.cpp

```diff
@@ -118,7 +119,14 @@ bool GLFont::load(const char* file)
     glTexImage2D(GL_TEXTURE_2D, 0, GL_ALPHA8, texWidth, texHeight, 0, GL_ALPHA, GL_UNSIGNED_BYTE, bitmap);
     glBindTexture(GL_TEXTURE_2D, 0);
 
+#ifdef __GNUC__
+    #pragma GCC diagnostic push
+    #pragma GCC diagnostic ignored "-Wold-style-cast"
+#endif
     if (!m_vboID && GLEW_VERSION_1_5)
```

Comment:
If the GLEW_VERSION_1_5 is causing cast warnings, wrap it up into a function, so you won't need to spam preprocessor directives for each use location.